src/xenia/kernel/objects/xthread.h

```diff
@@ -156,6 +160,8 @@ class XThread : public XObject {
   uint32_t pcr_address_;
   uint32_t thread_state_address_;
   cpu::ThreadState* thread_state_;
+  bool guest_thread_;  // Launched into guest code?
```

Comment:
Hah I gotcha - the question mark make me read that as 'Maybe it means it was launched into guest code? *shrug*' :P